test/groups.yaml

```diff
@@ -41,6 +41,24 @@ prometheus:
     - "opsportal"
     - "dispatch"
 
+# ------------------------------------------------------------------------------
+# Kommander
+#
+# All Kommander related addons should be tested as a part of this group
+# ------------------------------------------------------------------------------
+#    "kommander": {"opsportal", "cert-manager", "defaultstorageclass-protection", "localvolumeprovisioner", "traefik", "traefik-forward-auth", "dex", "kommander"},
```

Comment:
the comment and the list don't match. Is this intentional?

Reply:
definitely a NIT, sorry meant to remove that!